src/app/games/tafl/TaflState.ts

```diff
@@ -37,4 +38,13 @@ export class TaflState extends GameStateWithTable<TaflPawn> {
         return this.getHeight();
     }
 
+    public hasOwnerAt(coord: Coord, owner: PlayerOrNone): boolean {
+        const optional: MGPOptional<TaflPawn> = this.getOptionalPieceAt(coord);
+        if (optional.isPresent()) {
+            return optional.get().getOwner().equals(owner);
+        } else {
+            return false;
+        }
+    }
+
```

Comment:
j'ai l'impression qu'il y a *beaucoup* de nouvelles méthodes de ce genre, qui pourraient pas mal se faire en une ligne avec un `this.pieceAtSatisfiesPredicate((p: Optional<Piece>) => p.isPresent() && p.get().getOwner().equals(owner)))`

Reply:
Ouais après "en une ligne" c'pas ma def de "c'est mieux" haha